ssi-jws/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
-default = ["secp256k1", "secp256r1", "ed25519", "rsa", "eip", "ssi-jwk/ripemd-160"]
+default = ["secp256k1", "secp256r1", "ed25519", "rsa", "eip", "ssi-jwk/ripemd-160", "blake2b_simd"]
```

Comment:
should we ever not use simd?

Reply:
Ha, thanks past me, for once
```
// We will be able to use the blake2 crate directly once it allow 32B output
```

Which is a piece of work that I might be able to do now, it was tied to #385

Reply:
Oh sorry I misread your message, I guess it should be a dependency of other dependencies instead, I'll address it with the above